### PR TITLE
RPG Fixes

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request

Balances the HEDP round to be more anti-armor, less Fuck You Get Gibbed

Makes the Hammer have no safety.

Makes weapons with no bolt un-rackable. Hides the rack text so you don't get a prompt to do it
Restores old "use hand on boltless weapon" behavior to weapons with no bolt

## Why It's Good For The Game
Balance + Usability Change

## Changelog

:cl:

balance: Hammer RPGs should no longer annihilate you completely. Just partially.
fix: Weapons with no bolt can no longer be racked. Use your hand on them instead

/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic.dm [hunk 1]
```diff
@@ -270,30 +270,35 @@
...
-			CB.forceMove(drop_location())
+				var/angle_of_movement =(rand(-3000, 3000) / 100) + dir2angle(turn(user.dir, 180))
+				CB.AddComponent(/datum/component/movable_physics, _horizontal_velocity = rand(350, 450) / 100, _vertical_velocity = rand(400, 450) / 100, _horizontal_friction = rand(20, 24) / 100, _z_gravity = PHYSICS_GRAV_STANDARD, _z_floor = 0, _angle_of_movement = angle_of_movement, _bounce_sound = CB.bounce_sfx_override)
 
-			var/angle_of_movement =(rand(-3000, 3000) / 100) + dir2angle(turn(user.dir, 180))
-			CB.AddComponent(/datum/component/movable_physics, _horizontal_velocity = rand(350, 450) / 100, _vertical_velocity = rand(400, 450) / 100, _horizontal_friction = rand(20, 24) / 100, _z_gravity = PHYSICS_GRAV_STANDARD, _z_floor = 0, _angle_of_movement = angle_of_movement, _bounce_sound = CB.bounce_sfx_override)
+				num_unloaded++
+				SSblackbox.record_feedback("tally", "station_mess_created", 1, CB.name)
+			if (num_unloaded)
+				to_chat(user, "<span class='notice'>You unload [num_unloaded] [cartridge_wording]\s from [src].</span>")
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic.dm]:
```suggestion
				to_chat(user, span_notice("You unload [num_unloaded] [cartridge_wording]\s from [src]."))
```

File: code/modules/projectiles/guns/ballistic.dm [hunk 2]
```diff
@@ -270,30 +270,35 @@
...
+				playsound(user, eject_sound, eject_sound_volume, eject_sound_vary)
+				update_appearance()
+			else
+				to_chat(user, "<span class='warning'>[src] is empty!</span>")
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic.dm]:
```suggestion
				to_chat(user, span_warning("[src] is empty!"))
```

**Review by SomeguyManperson — COMMENTED** [review]
quick span macro check

**Comment by rye-rice** [comment]
update this pr in case its missing something before i merge it tommorow at 3am

**Pull request merged**